config/default.json

```diff
@@ -68,5 +68,8 @@
         "type": "Ingestion_Swap_Update"
       }
     }
+  },
+  "polygonPartsWorker": {
```

Comment:
Get rid of the `polygonPartsWorker` scope, locate `maxAttempts` within `tasks` scope:
![image](https://github.com/user-attachments/assets/3101255e-1869-4342-b346-059f52b0f2ec)